gallery/src/styles/NxList/NxListPage.tsx

```diff
@@ -75,6 +75,13 @@ const NxListPage = () =>
             <td className="nx-cell">Modifier of <code className="nx-code">.nx-list__item</code></td>
             <td className="nx-cell">Use this when you want to have a button on the far right.</td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.selected</code></td>
+            <td className="nx-cell">Utility class that goes along
+              with <code className="nx-code">.nx-list__item</code></td>
```

Comment:
I'm getting lint errors on this line — `Closing tag of a multiline JSX expression must be on its own line`

Reply:
> I'm getting lint errors on this line — `Closing tag of a multiline JSX expression must be on its own line`

Fixed in 5c769c5f7e9a5d583ad85a3dc0ec2a679036466a